batchSystems/gridengine.py

```diff
@@ -69,7 +71,7 @@ def prepareQsub(cpu, mem):
                      "LD_LIBRARY_PATH=%s" % os.environ["LD_LIBRARY_PATH"]]
     reqline = list()
     if cpu is not None:
-        reqline.append("p="+str(cpu))
+        qsubline.extend(["-pe", "shm", str(int(cpu))])
```

Comment:
Toil already switched to `-pe` but it uses `-pe smp` instead of `-pe shm`. Do you think they are equivalent?